kyo-scheduler/jvm/src/main/scala/kyo/scheduler/regulator/Admission.scala

```diff
@@ -76,9 +91,12 @@ final class Admission(
     schedule: Task => Unit,
     nowMillis: LongSupplier,
     timer: InternalTimer,
-    config: Config = Admission.defaultConfig
+    config: Config = Admission.defaultConfig,
+    rotationWindow: Duration = Flag("admission.rotationWindowMinutes", 60).minutes
```

Comment:
What about 5 minutes? Any reason the scheduler would need such a long window?

Reply:
It's not much about the scheduler itself. On each new rotation window, a new set of ids are accepted. If a system is under pressure, the shift in incoming load to a new set of users can increase the pressure on shared resources. For example, data might need to be loaded from databases to caches and cache evictions can compound the pressure. The longer window favors stability. The goal is to avoid the same set of users always being penalized across outages, not necessarily provide user-level fairness during a specific outage. It's always possible to tune the config, though.